apps/frontend/app/home/home.module.css

```
@@ -1,20 +1,242 @@
-/* ========================= Reusable styles - START ===================================== */
-/* Add rules that can be reused across sections here */
-/* ========================= Reusable styles - END ===================================== */
-
-/* ========================= Header section styles - START ===================================== */
-/* TODO: Add header rules here. The first rule has been partially set up for you */
-
 .header {
   color: var(--white);
   display: flex;
+  flex-direction: column;
+  gap: calc(var(--space-scale) * 2);
```

Stick to the variables in the design system

```suggestion
  gap: calc(var(--spacer-2))
```